clients/imodels-client-authoring/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Change Log - @itwin/imodels-client-authoring 
+
+## 2.0.0
+
+Breaking changes:
+- Removed `FileHandler` interface and default `AzureFileHandler` implementation. Storage operations now use `@itwin/object-storage-core` and `@itwin/object-storage-azure` packages.
```

Comment:
Could be useful to link these packages.

Reply:
Agree, added links: https://github.com/iTwin/imodels-clients/pull/126/commits/4cef427e9447e8df88b3aed3d16180269dbb9692